crates/uv/src/commands/python/list.rs

```diff
@@ -124,6 +125,7 @@ pub(crate) async fn list(
                 EnvironmentPreference::OnlySystem,
                 python_preference,
                 cache,
+                current_dir()?.as_path()
```

Comment:
This command does support the `--project` flag, I guess we should respect it? I'm not sure it matters here but it could bite someone later?